.github/workflows/release.yaml

```diff
@@ -23,3 +23,46 @@ jobs:
           args: release --clean
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  build-and-push:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      id-token: write
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Setup QEMU
+        uses: docker/setup-qemu-action@v3
+      - name: Setup Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: ${{ secrets.ROLE_TO_ASSUME }}
+          role-session-name: "mackerel-sql-metric-collector:${{ github.run_id }}-${{ github.run_number }}"
+          aws-region: ap-northeast-1
+      - name: Login to Public ECR
```

Comment:
[nit] [aws-actions/amazon-ecr-login](https://github.com/aws-actions/amazon-ecr-login) provides equivalent functions in a short setting and outputs such as registry name. 

```
uses: aws-actions/amazon-ecr-login@v2
with:
  registry-type: public
```

Reply:
`aws-actions/amazon-ecr-login` automatically retrieves and outputs the registry name based on AWS credentials.
This may be useful for private registries that vary by AWS account ID or region.
However, the public ECR name is unique in `public.ecr.aws`, so there seems no particular need to use it.